src/SWP/Bundle/FixturesBundle/DataFixtures/ORM/LoadWidgetsData.php

```diff
@@ -26,11 +28,14 @@ class LoadWidgetsData extends AbstractFixture implements FixtureInterface, Order
      */
     public function load(ObjectManager $manager)
     {
+        /** @var TenantInterface $tenant */
```

Comment:
the fixture is container aware so it allows you to use other services like the repository. Note that the widgets are stored using doctrine orm and tenants are stored in phpcr - the reference here will not work.